website/docs/docs/use-dbt-semantic-layer/tableau.md

```diff
@@ -59,12 +59,14 @@ Visit the [Tableau documentation](https://help.tableau.com/current/pro/desktop/e
 ## Things to note
 
 - All metrics use the "SUM" aggregation type, and this can't be altered. The dbt Semantic Layer controls the aggregation type and it is intentionally fixed. Keep in mind that the underlying aggregation in the dbt Semantic Layer might not be "SUM" (even though "SUM" is Tableau's default).
-- Tableau surfaces all metrics and dimensions from the dbt Semantic Layer on the left-hand side. Note, that not all metrics and dimensions can be combined with one another. You will receive an error message if a particular dimension cannot be sliced with a metric (or vice versa). 
+-  In the "ALL" data source, Tableau surfaces all metrics and dimensions from the dbt Semantic Layer on the left-hand side. Note, that not all metrics and dimensions can be combined with one another. You will receive an error message if a particular dimension cannot be sliced with a metric (or vice versa). For smaller slices of data that can be joined together, you can access saved queries.
    - To display available metrics and dimensions, dbt Semantic Layer returns metadata for a fake table with the dimensions and metrics as 'columns' on this table. Because of this, you can't actually query this table for previews or extracts. 
-   - Since this is treated as a table, dbt Semantic Layer can't dynamically change what is available. This means we display _all_ available metrics and dimensions even if a particular metric and dimension combination isn't available. 
-   
 - Certain Table calculations like "Totals" and "Percent Of" may not be accurate when using metrics aggregated in a non-additive way (such as count distinct)
 - In any of our Semantic Layer interfaces (not only Tableau), you must include a [time dimension](/docs/build/cumulative#limitations) when working with any cumulative metric that has a time window or granularity.
+- We support calculated fields to the extent that you are using them to create a parameter filter or dynamically select metrics and dimensions, but other calculated field functions are not supported.
```

Comment:
This sounds super awkward. @aiguofer trying to figure out if I've captured the gist of it.

Reply:
what about:

> We can support calculated fields for creating parameter filters or dynamically selecting metrics and dimensions. However, other calculated field functions are not supported.

Reply:
I like Mirna's suggestion but I would change `calculated field functions` to `uses of calculated fields`.

I wonder if we want to mention something like "if you have use cases for calculated fields that are not currently covered, please reach out so we understand the use case to see if we can support it".

There's probably a lot of use cases that we're not even aware of that might be totally legitimate and doable.

Reply:
That's a good idea! @mirnawong1 the "Things to note section" is getting kind of large. I'm going to try and break it up a bit and add these changes